inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -215,6 +221,7 @@ static void showUsage() {
     std::cout << "    -t                        " << execution_time_message << std::endl;
     std::cout << "    -progress                 " << progress_message << std::endl;
     std::cout << "    -shape                    " << shape_message << std::endl;
+    std::cout << "    -layout                   " << layout_message << std::endl;
```

Comment:
can we have request for output layout? `compile_tool` has:
```
    -ip                          <value>     Optional. Specifies precision for all input layers of the network.
    -op                          <value>     Optional. Specifies precision for all output layers of the network.
    -iop                        "<value>"    Optional. Specifies precision for input and output layers by name.
                                             Example: -iop "input:FP16, output:FP16".
                                             Notice that quotes are required.
                                             Overwrites precision from ip and op options for specified layers.
    -il                          <value>     Optional. Specifies layout for all input layers of the network.
    -ol                          <value>     Optional. Specifies layout for all input layers of the network.
    -iol                        "<value>"    Optional. Specifies layout for input and output layers by name.
                                             Example: -iol "input:NCHW, output:NHWC".
                                             Notice that quotes are required.
                                             Overwrites layout from il and ol options for specified layers.
```
May be we can have the same notation which is more future proff?

Reply:
@ilya-lavrenov Format of new parameter is aligned with existed `shape` parameter. For example:
```
-shape "image_info[2,3],image_tensor[2,3,200,200]" -layout "image_info[NC],image_tensor[NCHW]"
```

I think it is better to be aligned inside one tool.